BraveRewardsUI/Settings/Wallet/WalletDetailsViewController.swift

```diff
@@ -52,4 +54,30 @@ class WalletDetailsViewController: UIViewController, RewardsSummaryProtocol {
     let container = PopoverNavigationController(rootViewController: controller)
     present(container, animated: true)
   }
+  
+  func setupLedgerObservers() {
+    ledgerObserver.balanceReportUpdated = { [weak self] in
+      guard let self = self, self.isViewLoaded else {
+        return
+      }
+      let rows = self.summaryRows.map({ row -> RowView in
+        row.isHidden = true
```

Comment:
Why are you setting them all to hidden here?

Reply:
To animate an item in stack view we have to toggle isHidden

Reply:
So you're hiding all the current summary rows… but then animating them hidden below to then animate them to visible after? While not accessing `summaryRows` again? I'm confused

Reply:
Ah, I see, you're hiding the rows that are going to be set below. I see now. Is there a reason why you don't just do 

```
// Update rows but set them hidden to be animated
self.detailsView.activityView.rows = self.summaryRows.map { $0.isHidden = true; return $0 }
```

Would probably alleviate confusion

Reply:
Yeah will update :)